docs/website/docs/tutorial/sql_database.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Load data from a SQL database
+description: How to extract data from a SQL Database using dlt's SQL Database core source
+keywords: [sql connector, sql database pipeline, sql database]
+---
+
+This tutorial will show how you can use `dlt` to connect data from a SQL Database (PostgreSQL, MySQL, Microsoft SQL Server, Oracle, IBM DB2 etc.) into any `dlt`-compatible destination (Postgres, BigQuery, Snowflake, DuckDB etc.).  
+  
+To make it easy to reproduce, we will be loading data from the [public MySQL RFam database](https://docs.rfam.org/en/latest/database.html) into a local DuckDB instance.  
+
+## What you will learn
+
+- How to set up and configure a basic SQL database pipeline
+- How to implement "append", "replace", and "merge" loading strategies
+- How to load data incrementally
+
+## 0. Prerequisites
+
+- Python 3.9 or higher installed
+- Virtual environment set up
+- `dlt` installed. Follow the instructions in the [installation guide](../reference/installation) to create a new virtual environment and install `dlt`.
+
+## 1. Create a new `dlt` project
+
+Initialize a new `dlt` project in your current working directory using the `dlt init` command:
+
+```
+dlt init sql_database duckdb
+```
+
+This is a handy CLI command that creates files and folders required for a SQL Database to DuckDB pipeline. You can easily replace `duckdb` with any other [supported destinations](../dlt-ecosystem/destinations).  
+
+After running this command, your project will have the following structure:
+
+```txt
```

Comment:
i would add lines to show the structure, like this 
<img width="370" alt="Screenshot 2024-09-10 at 5 33 03 PM" src="https://github.com/user-attachments/assets/f7559f8e-049b-44d3-9e37-dd7f429d84c9">